frontend/src/screens/GroupPage/GroupPage.css

```diff
@@ -7,3 +7,51 @@
   transform: scale(1.05);
   cursor: pointer;
 }
+
+.group-page-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+}
+
+.group-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 0.8;
+  margin-bottom: 5%;
+  width: 100%;
+}
+
+.group-header {
+  width: 75vw;
+  max-width: 75vw;
+  margin: 0 auto;
+  position: relative;
+}
+
+.header-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  position: relative;
+}
+
+.leave-group-icon {
+  position: absolute;
```

Comment:
ngl i dont like position absolute did you check if this works for all screen sizes